pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/jstl/JstlEvaluator.java

```diff
@@ -81,6 +81,19 @@ private void registerFunctions() {
             "timestampAdd",
             JstlFunctions.class.getMethod(
                 "timestampAdd", Object.class, Object.class, Object.class));
+    this.expressionContext
+        .getFunctionMapper()
+        .mapFunction(
+            "fn",
+            "decimal",
+            JstlFunctions.class.getMethod("toBigDecimal", Object.class, Object.class));
+
+    this.expressionContext
+            .getFunctionMapper()
+            .mapFunction(
+                    "fn",
+                    "decimalFromDouble",
```

Comment:
EL doesn't support methods overloads, other names/suggestions are welcome (maybe `decimal(double, null)`? Because the coercion rules of the first arg will depend on the second, I thought it could be confusing to understand the behavior, hence separate methods).

Reply:
or maybe `decimalFromNumber`

Reply:
Yes, decimalFromNumber is better.
Sorry, only saw this now.